Gordon360/Services/ContentManagementService.cs

```diff
@@ -26,12 +30,56 @@ public ContentManagementService(UnitOfWork unitOfWork)
         /// </summary>
         /// <returns>If found, returns a set of SliderViewModel's, based on each slide entry in the db. 
         /// If not returns an empty IEnumerable.</returns>
-        public IEnumerable<SliderViewModel> GetSliderContent()
+        public IEnumerable<SliderViewModel> DEPRECATED_GetSliderContent()
         {
-           var query =  _unitOfWork.SliderRepository.GetAll();
-           var result = query.Select<C360_SLIDER, SliderViewModel>(x => x);
+            var query = _unitOfWork.SliderRepository.GetAll();
+            var result = query.Select<Slider_Images, SliderViewModel>(x => x);
+
+            return result;
+        }
+
+        /// <summary>
+        /// Retrieve all banner slides from the database
+        /// </summary>
+        /// <returns>An IEnumerable of the slides in the database</returns>
+        public IEnumerable<Slider_Images> GetBannerSlides() => _unitOfWork.SliderRepository.GetAll();
 
-           return result;
+        /// <summary>
+        /// Inserts a banner slide in the database and uploads the image to the local slider folder
+        /// </summary>
+        /// <param name="slide">The slide to add</param>
+        /// <returns>The inserted slide</returns>
+        public Slider_Images AddBannerSlide(BannerSlidePostViewModel slide)
+        {
+            string fileName = slide.Title + ".jpg";
```

Comment:
Is JPG the only file type that admins are able to upload?

It's probably better to have this pulled dynamically from the base64 string.